contributor_guide.md

```diff
@@ -0,0 +1,137 @@
+# Contribution Guide
+
+## Ways to contribute
+
+We welcome contributions in many forms;  
+- Share a use case, example or idea for a new tutorial
+- Submit a new tutorial
+- Update an existing tutorial
+- Submit bug reports and feature request
+- Fix typos
+- Add documentation
+
+If you would like to contribute, check the _Issues_ by clicking on the _Issues_ tab.
+Someone might already have a similar idea.  If not, then create a new issue and submit your sugestion.
+
+## How to contribute
+
+If you are not part of NSIDC please follow a __Forking Workflow__.  If you are part of NSIDC, you can follow a __Forking Workflow__ or follow a standard __Git Workflow__.
+
+We follow the __git workflow__ described [here](https://www.asmeurer.com/git-workflow/).
+This is a __Forking Workflow__.  This is fundamentally different from other workflows in
```

Comment:
As a GitHub newbie I find these lines a little confusing due to the multiple mentions of a git workflow and a forking workflow. If you are part of NSIDC it is not clear to me what a standard git workflow is, and the following sentence that says 'we follow the git workflow...' and then it goes on to describe a forking workflow is mostly what is confusing me. Reading through the rest of the guide it is clear a Forking workflow is being described here, but perhaps to avoid confusion in the beginning we should just refer to one workflow for both internal and external contributors and use one term for it?